inference-engine/thirdparty/clDNN/src/gpu/kernels_cache.cpp

```diff
@@ -435,6 +438,17 @@ void kernels_cache::build_all() {
         std::lock_guard<std::mutex> lock(_context.get_cache_mutex());
         _kernels_code.clear();
         _pending_compilation = false;
+#if (CLDNN_THREADING == CLDNN_THREADING_TBB)
+        arena.reset();
+#ifdef __unix__
+        malloc_trim(0);
+#endif
+#elif(CLDNN_THREADING == CLDNN_THREADING_THREADPOOL)
+        pool.reset();
+#ifdef __unix__
+        malloc_trim(0);
```

Comment:
Could you please add a comment here why malloc_trim is needed? 
And please check the impact on first inference latency, as malloc_trim seems to be heavy operation.

Reply:
Added comments. Also I will check for the inference latency for some networks, but it would be better to check with dlBenchmark for better coverarage..